src/tm2/tm-device.cpp

```diff
@@ -951,7 +951,7 @@ namespace librealsense
         }
         else
         {
-            LOG_WARNING("Dropped frame. alloc_frame(...) returned nullptr");
+            LOG_INFO("Dropped frame. alloc_frame(...) returned nullptr");
```

Comment:
In what scenario would `alloc_frame` return a nullptr that doesn't justify a warning?

Reply:
The warning occurs whenever frames acquired by users are not released in a timely manner.
We observe it some OpenGl-enabled demos where the default frames queue allocated in the SDK (size=16) is probably not enough to compensate for 3D scene render.

Reply:
The main problem is that this is a negative feedback loop - frame drops cause console output, slowing the app and causing more frame drops.
A better solution that we discussed for a while (I think @aangerma tried implementing at some point) is for processing blocks to implement notification extension (like the sensor does) and manage some basic internal statistics. This way, we can avoid spamming the output with such warnings...

Reply:
@ev-mp Should we make the frame queue deeper for imu data which is small but high rate?